app/assets/javascripts/polaris_view_components/autocomplete_controller.js

```diff
@@ -3,7 +3,7 @@ import { get } from '@rails/request.js'
 import { debounce } from './utils'
 
 export default class extends Controller {
-  static targets = ['popover', 'input', 'results', 'option', 'emptyState']
```

Comment:
Because the template is cloned and appended to the body these targets no longer work. I've tried to recreate these using Stimulus' naming and some get methods.